network/src/application/interface.rs

```diff
@@ -80,15 +82,70 @@ pub trait NetworkInterface {
 }
 
 #[derive(Clone)]
-struct MultiNetworkSender<
+pub struct SingleNetworkSender<
```

Comment:
maybe some docs here?

Reply:
Yeah, some documentation around `MultiNetworkSender` and `SingleNetworkSender` would provide more clarity for noobs.

For example, being the noob that I am, one of the questions I currently have is whether or not a `MultiNetworkSender` could be implemented using an internal set of `SingleNetworkSenders`? I suspect this is not the case, but the documentation might help me understand why/why not 😄

Reply:
Removed